src/test/java/net/unicon/lti/utils/lti/LtiOidcUtilsTest.java

```diff
@@ -84,8 +82,6 @@ public void testGenerateState() {
             assertEquals("client-id", ltiTokenClaims.getAudience());
             assertEquals("nonce-value", ltiTokenClaims.getId());
             assertEquals("test-iss", ltiTokenClaims.get("original_iss"));
-            assertEquals("login-hint", ltiTokenClaims.get("loginHint"));
-            assertEquals("lti-message-hint", ltiTokenClaims.get("ltiMessageHint"));
```

Comment:
Shouldn't we also add an assertion that the "nonce" is getting set to "nonce-value"?

Reply:
Commented in chat